src/evaluationUtils/PositionedError.ts

```diff
@@ -0,0 +1,19 @@
+import { SourceRange } from '../expressions/debug/StackTraceGenerator';
```

Comment:
NIT: We do tend to add in some white lines for readability. e.g. between the import and export and before the constructor.